docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -6337,6 +6337,300 @@ UPDATE persnl.job
 SET jobdesc = RIGHT (jobdesc, 12);
 ```
 
+<<<
+[[rollup_function]]
+== ROLLUP Function
+
+The ROLLUP function calculates multiple levels of subtotals aggregating from right to left through the comma-separated list of columns, and provides a grand total. It is a an extension to the `GROUP BY` clause and can be used with `ORDER BY` to sort the results.
```

Comment:
There is no "ORDER BY ROLLUP" syntax. But it looks like there are functions such as GROUPING that can refer to whether a column is used as a grouping column in a rollup result row, so one can order the detail vs. the summary rows. There's also a small typo here "...is a an..." Possible wordsmith for the last sentence: "It is an extension to the 'GROUP BY' clause. Related features such as the GROUPING function can be used with 'ORDER BY' to control the placement of summary results."

Reply:
Thanks Dave, your comment has been incorporated :)